org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/uriresolver/CacheResourcesManagerTest.java

```diff
@@ -89,7 +89,7 @@ public void testUnavailableCache() throws Exception {
 	public void testAvailableCache() throws Exception {
 		FileServer server = new FileServer();
 		server.start();
-		String uri = server.getUri("/dtd/web-app_2_3.dtd");
+		String uri = server.getUri("/dtd/base.dtd");
```

Comment:
how is that related to this PR?

Reply:
Yes you are right, I removed this update.